test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/StringTokenizerTest.scala

```diff
@@ -0,0 +1,96 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.javalib.util
+
+import java.util.StringTokenizer
+
+import org.junit.Assert._
+import org.junit.Test
+
+import org.scalajs.testsuite.utils.AssertThrows._
+
+class StringTokenizerTest {
+
+  @Test def constructor_with_delim(): Unit =  {
+    val st = new StringTokenizer("This:is:a:test:String", ":")
+    assertEquals(5, st.countTokens())
+    assertEquals("This", st.nextElement())
+  }
+
+  @Test def constructor_with_delim_with_returnDelims(): Unit = {
+    val st = new StringTokenizer("This:is:a:test:String", ":", true)
+    assertEquals(9, st.countTokens())
+    st.nextElement()
+    assertEquals(8, st.countTokens())
+    assertEquals(":", st.nextElement())
+  }
+
+  @Test def countTokens(): Unit = {
+    val st = new StringTokenizer("This is a test String")
+    assertEquals(5, st.countTokens())
+  }
+
+  @Test def hasMoreElements(): Unit = {
+    val st = new StringTokenizer("This is a test String")
+    st.nextElement()
+    assertTrue(st.hasMoreElements())
+    st.nextElement()
+    st.nextElement()
+    st.nextElement()
+    st.nextElement()
+    assertFalse(st.hasMoreElements())
+  }
+
+  @Test def hasMoreTokens(): Unit = {
+    val st = new StringTokenizer("This is a test String")
+    var counter = 0
+    while (counter < 5) {
+      assertTrue(st.hasMoreTokens())
+      st.nextToken()
+      counter += 1
+    }
+    assertFalse(st.hasMoreTokens())
+  }
+
+  @Test def nextElement(): Unit = {
+    val st = new StringTokenizer("This is a test String")
+    assertEquals("This", st.nextElement())
+    assertEquals("is", st.nextElement())
+    assertEquals("a", st.nextElement())
+    assertEquals("test", st.nextElement())
+    assertEquals("String",  st.nextElement())
+
+    assertFalse(st.hasMoreTokens())
```

Comment:
```suggestion
    assertFalse(st.hasMoreElements())
```
would be more consistent